invenio_vocabularies/contrib/subjects/config.py

```diff
@@ -40,7 +46,7 @@ class SubjectsSearchOptions(SearchOptions):
         ),
         "subject": dict(
             title=_("Name"),
-            fields=["subject_sort"],
+            fields=["indexed_at"],
```

Comment:
minor: we should keep `subject_sort` as the sorting field (it's still present in the schema and is a required field).